graphql-async/test/jbuild

```diff
@@ -0,0 +1,10 @@
+(jbuild_version 1)
+
+(executables
+ ((libraries (graphql_async alcotest async))
```

Comment:
OK this is good now. Note that for async we depend on async_kernel for the dependency itself and async_unis is only required for tests.